README.md

```diff
@@ -64,17 +64,11 @@ Key:
 
 ## Guides
 
-### User Guide
-
 See our documentation on [kubeedge.io](https://kubeedge.io).
 
 **Quick Start** - Install KubeEdge with [keadm](./docs/setup/keadm.md).
 
-Try some examples of KubeEdge on [examples](https://github.com/kubeedge/examples).
-
-### Developer Guide
-
-Take a look at our [development guide], If you are interested in building and contributing KubeEdge.
+To learn deeply about KubeEdge, try some examples on [examples](https://github.com/kubeedge/examples).
```

Comment:
I think encouraging users to contribute is necessary here.

Reply:
So how about moving `CONTRIBUTE.md` here?